Jenkinsfile

```diff
@@ -13,17 +13,6 @@ def sonarQubeEnv = 'SonarQube'
 def sonarScanner = 'SonarScanner'
 def timeoutInMinutes = 5
 
-def getExtraCommands(pr) {
```

Comment:
The reason the `getExtraCommands` function was introduced was to move the bespoke steps out of the main body of the pipeline so all pipelines take the same format, aside from parameters passed in.

The idea being it would be easier to move to a boilerplate pipeline step when the shared library is refactored.
Inlining the function feels like a regression, as it will need to be pulled back out once the library is updated.

Reply:
Because we're now dependent on a variable from the library, ie `containerTag` that approach wouldn't work. Although we could pass it as a parameter like `pr` but that doesn't seem scalable.